autosubmit/autosubmit.py

```diff
@@ -97,23 +97,17 @@
 # noinspection PyUnusedLocal
 
 
-def signal_handler(signal_received, frame):
+def signal_handler():
     """
     Used to handle interrupt signals, allowing autosubmit to clean before exit
-
-    :param signal_received:
-    :param frame:
     """
     Log.info('Autosubmit will interrupt at the next safe occasion')
     Autosubmit.exit = True
 
 
-def signal_handler_create(signal_received, frame):
+def signal_handler_create():
```

Comment:
These args need to be here. They are used by the `signal.signal`, https://docs.python.org/3/library/signal.html#signal.signal

![image](https://github.com/user-attachments/assets/a09f1eeb-8250-4592-81e5-150eb97131d1)

This means that if we remove it, we have something akin to:

![image](https://github.com/user-attachments/assets/48935dd8-b2da-477d-bacf-bdc5d7ed1bf3)

Which causes a runtime error. So this needs to stay. We need to mark it as ignored by linters :+1: (PyCharm warns about it...)